src/queries/a11yState.ts

```diff
@@ -0,0 +1,68 @@
+import type { ReactTestInstance } from 'react-test-renderer';
+import { AccessibilityState } from 'react-native';
+import { makeQueries } from './makeQueries';
+import type {
+  FindAllByQuery,
+  FindByQuery,
+  GetAllByQuery,
+  GetByQuery,
+  QueryAllByQuery,
+  QueryByQuery,
+} from './makeQueries';
+
+export function matchObject<T extends Record<string, unknown>>(
+  prop: T | undefined,
+  matcher: T
+): boolean {
+  return prop
+    ? Object.keys(matcher).length !== 0 &&
+        Object.keys(prop).length !== 0 &&
+        !Object.keys(matcher).some((key) => prop[key] !== matcher[key])
```

Comment:
Using de Morgans law it seems easier to read. Is there any reason for favoring negated some over every?

```suggestion
        Object.keys(matcher).every((key) => prop[key] === matcher[key])
```

Reply:
done